src/lib/features/export-import-toggles/export-import-service.ts

```diff
@@ -690,6 +703,30 @@ export default class ExportImportService {
         const filteredTagTypes = tagTypes.filter((tagType) =>
             featureTags.map((tag) => tag.tagType).includes(tagType.name),
         );
+
+        const reducedFeatureDependencies = Object.values(
```

Comment:
can we use lodash groupBy to better express the intent without having to read the algorithm? One example is in the playground-service.ts